src/bidiMapper/domains/context/browsingContextProcessor.ts

```diff
@@ -291,6 +291,14 @@ export class BrowsingContextProcessor {
     return context.getOrCreateSandbox(target.sandbox);
   }
 
+  async process_script_addPreloadScript(
+    _params: Script.AddPreloadScriptParameters
+  ): Promise<Script.AddPreloadScriptResult> {
+    return {
+      script: '', // TODO(#293): Populate script.
```

Comment:
If you want to merge this PR before continue working on it, please throw a "Not implemented exception" here, so that not to confuse the users.